libs/logger.js

```diff
@@ -0,0 +1,22 @@
+import winston from 'winston';
+
+/**
+ * Log function that outputs console logs in json format.
+ *
+ * @param {('error'|'warn'|'info'|'http'|'verbose'|'debug'|'silly')} level Log level based on npm log level standard.
+ * @param {string} message   Log message.
+ * @param {string} requestId Request id to be able to filter logs on a single request.
+ * @param {string} errorCode Error code for locating log in code.
+ * @param {object} data      Custom data related to log.
```

Comment:
Comment suggest the param can be named customData?

Reply:
Can do that.

Reply:
or event customLogData. Then again all ting is data so I would exclude it from the name and maybe just name it customLog